### DISCUSSION TITLE
Photino future features poll. Please Vote!

### DISCUSSION THREAD

**Discussion by ottodobretsberger** [tryphotino/photino.NET#117] [Polls]
**Hello Photino community!**
We'd like to take a moment of your time and ask which feature you'd like to see implemented in Photino next. Please pick from the available options. If you have particular suggestions that you feel would deserve a higher priority, please comment below with your suggestion.

**Comment by endfix** [comment]
Similar method Window.DragMove from the application side for all OS

And thank you for the project development!

**Reply by jlshattuck** [reply]
Please consider adding support for chromeless windows and transparency.
ty

**Comment by artemiusgreat** [comment]
Something's missing, like interaction with the DOM elements from the external webpage loaded via `NavigateToUrl`
https://github.com/tryphotino/photino.NET/issues/58

**Comment by stevesobol** [comment]
I voted for native menus, but support for other languages, particularly Java, is important to me as well

**Comment by dameng324** [comment]
when ios will be supported,it's not on the plan, right?

**Comment by faudzanfadhli** [comment]
I voted for installers, and I need new properties to disabled view page source by pressing Ctrl+U. How to achieve it?

**Reply by MikeYeager** [reply]
@faudzanfadhli We're not sure this is possible as it seems to be baked into the browser controls. There is this article that tells you how to disable it in the registry: https://learn.microsoft.com/en-us/deployedge/microsoft-edge-policies#configurekeyboardshortcuts. We're not sure how to do it on Mac or Linux. We recently implemented a way to pass feature flags into the browser control initialization and there may be a setting you can use there, but we're not aware of it. Those startup flags are not well documented and they're different on each platform. If it's something you really need, please open an issue in the Photino.Native repo and perhaps the community can come up with a solution.

**Comment by AmirAtiaH** [comment]
create a discord server 👀

**Reply by ottodobretsberger** [reply]
At the moment, this is not planned. We simply do not have the resources to monitor and moderate efficiently.

**Comment by zyhhb** [comment]
android ios when will be supported?

**Comment by hewhocannotbetamed** [comment]
_actual_ documentation should be on this list.
